src/MinimumNodeService.h

```diff
@@ -51,7 +51,7 @@ class MinimumNodeService : public Service
   
   unsigned long lastHeartbeat = 0;
   byte heartbeatSequence = 0;
-  bool noHeartbeat = false;
+  bool noHeartbeat;
```

Comment:
Please keep the default initialisation. This reduces problems. Yes I know it will be set at startup but I prefer belt and braces.

Reply:
OK